src/group.ts

```diff
@@ -0,0 +1,6 @@
+import { Expression } from "./types";
+import { compile } from "./verbalexpressions";
+
+export default function group(...inputs: Expression[]) {
```

Comment:
It would be nice to have `group` support the following:

- `group('foo')` — functionally identical to `group.capturing`
- `group.capturing('foo')`
- `group.nonCapturing('foo')`